browser/flagr-ui/src/components/Flag.vue

```diff
@@ -246,12 +246,12 @@
                             </el-input>
                           </el-col>
                           <el-col :span="12">
-                            <el-input
-                              size="small"
-                              placeholder="{}"
-                              v-model="variant.attachmentStr">
-                              <template slot="prepend">Attachment </template>
-                            </el-input>
+                            <vue-json-editor
```

Comment:
I wonder if we can add some title or helper string before the json editor to indicate
- It's for variant attachment
- It only supports string => string

![image](https://user-images.githubusercontent.com/658840/58121541-88c21680-7bbc-11e9-82fe-85d6f8c1dd77.png)

I'm not good at UI design, so it's really up to you what to design for the attachment section. Maybe we can hide it with a button if it's empty.

Reply:
Alright, I will add more informative title.

Reply:
What do you think about the following?
![image](https://user-images.githubusercontent.com/13322290/58152670-c59e1400-7c75-11e9-8289-7fdee85fe13e.png)

Reply:
One more adjustment:
![image](https://user-images.githubusercontent.com/13322290/58162722-5fbc8700-7c8b-11e9-8bc4-fd70e290fde4.png)

Reply:
The variant id is important, so we can't remove it from the UI. Also the key is duplicated when expanded. Otherwise, lgtm.

Reply:
So I can stay with the second design. Is what do you think?
![image](https://user-images.githubusercontent.com/13322290/58152670-c59e1400-7c75-11e9-8289-7fdee85fe13e.png)